xbmc/guilib/Texture.cpp

```diff
@@ -493,3 +497,43 @@ bool CTexture::IsMipmapped() const
 {
   return m_mipmapping;
 }
+
+void CTexture::ConvertToBGRA()
+{
+  if (m_singleChUsage == SINGLE_CH_DEFAULT)
+    return;
+
+  unsigned char* oldPixels = m_pixels;
+  unsigned char* newPixels;
+
+  size_t size = GetPitch() * GetRows();
+  newPixels = static_cast<unsigned char*>(KODI::MEMORY::AlignedMalloc(size * 4, 32));
```

Comment:
I don't like uninitialized variables, because the memory is invalid between the lines where it's declared and defined, so instead we could do:

```c++
size_t size = GetPitch() * GetRows();

unsigned char* oldPixels = m_pixels;
unsigned char* newPixels = static_cast<unsigned char*>(KODI::MEMORY::AlignedMalloc(size * 4, 32));
```

Reply:
Done.

I hope I remember this in the future.